hack/update-dockerfile.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Script to insert two lines in Dockerfile after line 110
+
+set -e
+
+DOCKERFILE="Dockerfile"
+TEMP_FILE="Dockerfile.tmp"
+
+# Check if Dockerfile exists
+if [ ! -f "$DOCKERFILE" ]; then
+    echo "Error: $DOCKERFILE not found in current directory"
+    exit 1
+fi
+
+echo "Processing $DOCKERFILE..."
+
+# Insert the lines after line 110
+{
+    head -n 110 "$DOCKERFILE"
```

Comment:
Maybe let's search for a particular string here instead of a line number? In case the dockerfile gets changed until this PR is merged.